lib/index.css

```diff
@@ -16,14 +16,149 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
+
+.menu-top-line, 
+.menu-middle-line, 
+.menu-bottom-line,
+.menu-left-x,
+.menu-right-x {
+  border-bottom: 3px solid #ffffff;
+  transition: all 0.5s ease;
+}
+
+.menu-left-x {
+  transform: rotate(45deg);
+  top: 7px;
+  position: relative;
+}
+
+.menu-right-x {
+  transform: rotate(-45deg);
+  bottom: 5px;
+  position: relative;
+}
+
+.slide-pane {
+  transition: all 0.2s ease-in-out;
+}
+
+.slide-pane_from_left {
+  margin: 52px auto 0 0;
+}
+
+.slide-pane__header,
+.slide-pane__close {
+  display: none;
+}
+
+.slide-pane__content {
+  padding: 24px 0;
+}
+
+.app-menu {
+  margin: 0;
+  padding: 0;
+}
+
+.app-menu li {
+  padding: 1rem 2rem;
+  cursor: pointer;
+}
+
+.app-menu li img {
+  width: 25px;
+  margin-right: 2rem;
```

Comment:
My personal preference is 1rem

Reply:
We might want to have a call to go over only padding once we're done with this round of review

<img width="195" alt="Screen Shot 2021-09-14 at 3 01 43 PM" src="https://user-images.githubusercontent.com/86619099/133271912-35d82b18-35fc-4ddb-9ccc-0558594f36cf.png">